### PR TITLE
docs(README): HMR (`extracted-loader|css-hot-loader`)

### PR DESCRIPTION
**Pull request by sheerun**
Disclaimer: I'm an author of extracted-loader

### CODE REVIEW

**Comment by sheerun** [comment]
And by the way, I'd be happy if you decided to incorporate extracted-loader into extract-text-webpack-plugin behind a flag (e.g. `hotreload=true`)

**Comment by evilebottnawi** [comment]
@sheerun Thanks for PR. Feel free to PR. It would be great if `extract-text-webpack-plugin` will be support hot reload. I think we should this support without option, just out of box.

**Comment by sheerun** [comment]
This might be a breaking change for some people if they already use some kind of hot reloading solution. Are you sure you want patch change without any flag?

**Comment by evilebottnawi** [comment]
@sheerun This PR is patch, new PR with support hot reloading will have other label :+1: PR welcome

File: README.md [hunk 1]
```diff
@@ -219,6 +219,11 @@ plugins: [
 ]
 ```
 
+### Hot reloading
+
+The [extraced-loader](https://github.com/sheerun/extracted-loader) or [css-hot-reloader](https://github.com/shepherdwind/css-hot-loader) might be helpful.
```
Comment (michael-ciniawsky) [inline README.md]: `### Hot reloading`  => ```### `HMR` ```

```diff
- css-hot-reloader
+ css-hot-loader
```

**Review by michael-ciniawsky — CHANGES_REQUESTED** [review]

**Comment by alexander-akait** [comment]
DEPRECATED
Please use: https://github.com/webpack-contrib/mini-css-extract-plugin

**Pull request closed without merging**